lib/src/rx_command_executable.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/widgets.dart';
+import 'package:rx_command/rx_command.dart';
+
+/// Wrap your [Widget] into [RxCommandExecutable] widget to get an ability to listen for events
```

Comment:
I'm not clear what you would do with such a widget honestly. I think if you need something like that you probably also need a stateful widget and register your own CommandListener. With this widget you can't pass any values to the build method as its fixed.
Perhaps you can share your thoughts on this with me.